README.md

```diff
@@ -1 +1,17 @@
-# Portifolio-mobile-menu
\ No newline at end of file
+# curriculum-cr-training-mobile-menu
+
+
+## Author
```

Comment:
- [ ] It would be better if you add more sections to your readme file using [this template](https://github.com/microverseinc/readme-template).
- [ ] Make sure to mention Figma design in the Acknowledgments section